### PR TITLE
New Vulp hairstyle

### PR DESCRIPTION
**Pull request by FishStix4Dayz**
I created a new Vulpkanin hairstyle called raine. It works fine i've tested it and everything.
🆑: 
add : Raine hairstyle for vulpkanin.
/🆑

screenshot

yes, i got permission to use the screenshot

### CODE REVIEW

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1292,6 +1292,12 @@
 	name = "Rough-Cropped Mane"
 	icon_state = "rough"
 
+/datum/sprite_accessory/hair/vulpkanin/vulp_hair_raine
+	name = "Raine"
+	icon_state = "vulp_hair_raine"
+	species_allowed = list("Vulpkanin")
```
Comment (variableundefined) [inline code/modules/mob/new_player/sprite_accessories.dm]: Redundant, declared already in parent

Comment (KasparoVy) [reply code/modules/mob/new_player/sprite_accessories.dm]: Hey, we worked hard to eliminate unnecessary `species_allowed`! :wink:

**Comment by variableundefined** [comment]
This is your first PR. Since it isn't a fix (And even then, we still recommend doing so for fixes at time), add in a changelog in this format

> :CL:
> add: Raine hairstyle for vulpkanin
> /:CL: 

It will show up as this:

> 🆑:
> add: Raine hairstyle for vulpkanin
> /🆑

**Comment by taukausanake** [comment]
Holy shit that's some nice hair. Great sprite work!

Looks like someone's a fan of Two Kinds :^)

**Comment by KasparoVy** [comment]
Fischbach ftw
Both of em

**Comment by FishStix4Dayz** [comment]
sooo, everything is good???

**Comment by KasparoVy** [comment]
You still need to addresd the critique about `sprite_accessories.dm` line `1298` but otherwise it seems fine

File: code/modules/mob/new_player/sprite_accessories.dm (hunk 1 above)
Comment (taukausanake) [reply code/modules/mob/new_player/sprite_accessories.dm]: That species_allowed list annoyed me for the longest time. When was it fixed? I can't remember

**Comment by FishStix4Dayz** [comment]
k, ill just delete that extra line of code

File: code/modules/mob/new_player/sprite_accessories.dm (hunk 1 above)
Comment (KasparoVy) [reply code/modules/mob/new_player/sprite_accessories.dm]: It was done in this mega-PR. Should definitely not be used as an example of atomic development :innocent: https://github.com/ParadiseSS13/Paradise/pull/5416

**Comment by TravellingMerchant** [comment]
It's a great sprite! Shame its vulp only though, I bet it would look great on humans too!

(I mean, technically Raine is human too)

**Comment by FishStix4Dayz** [comment]
yes yes i know, i could work on that but it will take some time, i'm also gonna have to fix the errors i have with the current style first.

**Comment by taukausanake** [comment]
You sure it work work for humans as it is? Looks like it might, aside from maybe the north-facing sprite

**Pull request merged**